scipy/interpolate/tests/test_rgi.py

```diff
@@ -864,6 +905,9 @@ def test_complex(self, method):
         v2r = interpn(points, values.real, sample, method=method)
         v2i = interpn(points, values.imag, sample, method=method)
         v2 = v2r + 1j*v2i
+
+        # atol = 5e-6 if method == "quintic" else 1e-15
+        # assert_allclose(v1, v2, atol=atol)
```

Comment:
Is there a reason these lines are still uncommented?

Reply:
Thanks! Removed.